Resolve conflicts from main (#242)
LiskHQ/lisk-docs: introduction.md

This token is used to pay the fees associated with sending transactions on the network. Each token also allows the holder to vote for delegates that secure the network within the Lisk DPoS (Delegated Proof of Stake) consensus protocol.

LSK tokens can be acquired through a number of ways, such as purchasing them from an exchange or from another individual using Fiat or Bitcoin (BTC).

## The Lisk Protocol

The Lisk Protocol is a formal set of rules of the Lisk blockchain.

It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.

> The Lisk protocol is constantly evolving. The discussions about planned changed take place at [research.lisk.io](https://research.lisk.io).

Reviewer: `The discussions about planned changed...`
->
The discussions about the planned changes.

The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).

The protocol defines these rules without a focus on concrete implementation, and it is, therefore, possible to implement it in any compatible programming language.

> Currently, the official implementations of the Lisk protocol are [Lisk Core](../lisk-core/introduction.md), [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) and [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md), which are written in Javascript and Typescript.

Reviewer: I assume that the path for these three links is going to change (now we don't have "lisk-sdk" folder in the docs repo. So please double check that they are correct because I can't from this file in this branch

Reviewer 2: We first can check this once everything is merged into `development`

This token is used to pay the fees associated with sending transactions on the network. Each token also allows the holder to vote for delegates that secure the network within the Lisk DPoS (Delegated Proof of Stake) consensus protocol.

LSK tokens can be acquired through a number of ways, such as purchasing them from an exchange or from another individual using Fiat or Bitcoin (BTC).

## The Lisk Protocol

> The Lisk Protocol is a formal set of rules of the Lisk blockchain.

Reviewer 3: ```suggestion
The Lisk Protocol is the formal set of rules of the Lisk blockchain.
```

> The Lisk Protocol is a formal set of rules of the Lisk blockchain.

Reviewer 3: ```suggestion
The Lisk Protocol is the formal set of rules of the Lisk blockchain.
```

> It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.

Reviewer 3: ```suggestion
It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```

Reviewer 3: Slightly modified the formulation and added links to the `blocks` and `transactions` section.

> It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.

Reviewer 3: ```suggestion
It defines rules for the [peer-to-peer communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```

> The Lisk protocol is constantly evolving. The discussions about planned changes take place at [research.lisk.io](https://research.lisk.io).

Reviewer 3: ```suggestion
The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Discussions about concrete proposed changes to the Lisk protocol take place openly and transparently at [Lisk Research](https://research.lisk.io).
```

> The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Discussions about changes to the Lisk protocol take place openly and transparently at [Lisk Research](https://research.lisk.io).

Reviewer 3: ```suggestion
The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0001.md) and discussions about possible LIPs take place openly and transparently at [Lisk Research](https://research.lisk.io).
```

> The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).

Reviewer 3: After a LIP comes into effect, i.e., the protocol changes have been adopted by the Lisk blockchain, the official [Lisk documentation](https://docs.lisk.io) will be updated accordingly.

> The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).

Reviewer 3: ```suggestion
After a LIP comes into effect, i.e., the protocol changes have been adopted by the Lisk blockchain, the official Lisk documentation will be updated accordingly.
```

> The protocol defines these rules without a focus on concrete implementation, and it is, therefore, possible to implement it in any compatible programming language.

Reviewer 3: I would maybe remove `compatible`, because I don't understand how a programming language can be compatible or not.

Reviewer 2: hm yeah, but it might not be 100% correct to say "in every programming language" as well.
There might be some turing incomplete languages, that cannot implement everything the protocol describes.

But beside this I imagine it's theoretically possible with every turing complete language.

Reviewer 3: That makes sense. What about `common` instead?

> The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0001.md) and discussions about possible LIPs take place openly and transparently at [Lisk Research](https://research.lisk.io).

Reviewer 3: ```suggestion
The Lisk protocol is evolving following the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0001.md) and discussions about possible LIPs take place openly and transparently at [Lisk Research](https://research.lisk.io).
```
